docs/COMMANDS.md

```diff
@@ -29,7 +29,19 @@ This will run a server that's accessible in the entire local network and shows t
 ## Unit testing
 
 ```Shell
+$ npm run lint
+```
+
+Checks .js files for any style errors.
```

Comment:
Could you change this to `Checks your JavaScript files for potential errors and suspicious usage of included functions` or something to that effect?

I think that's clearer for somebody who doesn't know what linting is and why you should lint your code